extensions/amp-delight-player/0.1/amp-delight-player.js

```diff
@@ -235,12 +235,13 @@ class AmpDelightPlayer extends AMP.BaseElement {
   createPlaceholderCallback() {
     const html = htmlFor(this.element);
     const placeholder = html`
-      <div placeholder><amp-img layout="fill"></amp-img></div>
+      <img placeholder referrerpolicy="origin" loading="lazy" />
     `;
 
     const src = `${this.baseURL_}/poster/${this.contentID_}`;
 
-    placeholder.firstElementChild.setAttribute('src', src);
+    placeholder.setAttribute('src', src);
+    this.applyFillContent(placeholder);
```

Comment:
Nit: Can we applyFillContent first? as a general rule we should always set src as the last thing on an image to prevent double downloads in Safari.